nixos/modules/services/networking/openvpn3.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.openvpn3;
+in
+{
+  ###### interface
+
+  options.services.openvpn3.client = {
```

Comment:
I've been using openvpn3 for a couple months now (based on the original PR package). It seems that openvpn3 is changing your DNS servers (via `/etc/resolv.conf` at runtime. I think it'd be a good default not to do that (if possible via a settings) for privacy reasons. 
Or *at least* document it.

Reply:
In my experience, a lot of VPNs override the default DNS server, and I think it's a common practice. At least, I had the same problem when connecting to the company's VPNs quite a few times. I think it's an intentional feature; furthermore, I think disabling it is impossible, and even if it was, it would break a lot of things.

The solution I see is improving the documentation. If you are connecting via Network Manager, there's a pretty simple solution: you can change the DNS backend to `systemd-resolved`, and it will enable conditional DNS. Most people want that, I suppose. 

That's, however, out of the scope of this PR. I would love to see discussion about the issue, though!

Reply:
Well you can use `openvpn3 config-manage --config XXX --dns-setup-disabled true` to avoid having this override. 
I agree with the fact that it may break things for people (and thus not having it by default). But I would prefer documenting it so that it doesn't come as a surprise for users :).